experiments/music/gather_eeg_tones.py

```diff
@@ -0,0 +1,67 @@
+import OpenSync
+import time
+import datetime
+import os
+from psychopy import prefs
+
+import os 
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+DATA_DIR = "{}/data/".format(dir_path)
+PREFS = {
+    # Hardware / display configuration
+    'audioLib': ['PTB', 'sounddevice', 'pyo', 'pygame'],
+    'audioLatencyMode': 4, # critical
+    'winType': ['pygame'],
+    # Setting to True makes text sharp, but causes problems with the dialogue
+    'highDPI': False,
+    'windowSize': (3840, 2160),
+    'monitorName': 'Blade13scaled',
+    'monitorWidth': 55,
+    'fullScreen': True,
+
+    # UI settings, colors, etc.
+    'windowColor': (1, 1, 1),
+    'textColor': (-1, -1, -1),
+}
+
+prefs.hardware['audioLib'] = PREFS.get('audioLib')
+prefs.general['winType'] = PREFS.get('winType')
+prefs.hardware['highDPI'] = PREFS.get('highDPI')
+prefs.saveUserPrefs()
+
+# this needs to happen after prefs are set
+from lib.psy import Psypy
+from lib.psyparticipant import display_participant_dialogue
+
+participant_id = display_participant_dialogue()
+
+xrunner = Psypy(conf = PREFS)
+
+
+# gonna go out on a limb here before even testing
```

Comment:
It's Python though so it might just not care - OpenSync might be Python-wrapped C++ of course but still quite a small chance